lib/cleaner.rb

```diff
@@ -0,0 +1,24 @@
+class Cleaner
+  attr_accessor :contents
+
+  def initialize
+    @file = "/Users/alexamsmyth/turing/1mod/projects/black_thursday/data/merchants.csv"
```

Comment:
Does this need to be the entire file name? I feel like this would only work locally and specifically for you, if that makes sense.. but I'm not sure

Reply:
great catch! I will correct this so its just the local path and will work on all computers.

Reply:
I have updated this in my latest push.